ecosystem/sep-0012.md

```diff
@@ -302,6 +340,41 @@ For example:
 }
 ```
 
+## Customer PUT Verification
+
+This endpoint allows servers to accept data values, usually confirmation codes, that verify a previously provided field via `PUT /customer`, such as `mobile_number` or `email_address`. Note that while fields such as `photo_proof_residence` or `notary_approval_of_photo_id` are verifications of other fields described in [SEP-9](#sep-0009.md), the server does not require the associated fields _before_ verification can be accomplished.
+
+### Request
+
+```
+PUT [KYC_SERVER || TRANSFER_SERVER]/customer/verification
+```
+
+Name | Type | Description
+-----|------|------------
+`id` | string | The ID of the customer as returned in the response of a previous `PUT` request.
+
+Data values used for verification should be included in the request using the SEP-9 attribute name plus `_verification`. All verification values must be of type `string`.
+
+```json
+{
+   "id": "391fb415-c223-4608-b2f5-dd1e91e3a986",
+   "mobile_number_verification": "2735021"
+}
+```
+
+### Response
+
+Success responses should return a `202 Accepted` status, and the `GET /customer` `status` of the field(s) being verified should be changed to `PROCESSING` or `ACCEPTED`.
```

Comment:
❓ What should the response body contain in the success case?

Reply:
This is actually undefined for DELETE, and `PUT` used to be undefined until we added the `id` attribute. We could return the field `status`?

Reply:
I think that sounds good.

Reply:
Actually, the request format allows for multiple field verifications to be sent, so a single status wouldn't work. I think returning the same response sent in `GET /customer` would make sense though.

Reply:
Sounds good to me.